packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -49,6 +51,18 @@ function addVNodeToChildLWC(vnode: VCustomElement) {
     ArrayPush.call(getVMBeingRendered()!.velements, vnode);
 }
 
+// [st]atic node
+function st(fragment: Element, key: Key): VStatic {
+    return {
+        type: VNodeType.Static,
+        sel: undefined,
+        key,
```

Comment:
I asked a question about this, but was never answered.

Reply:
https://github.com/salesforce/lwc/pull/2781#discussion_r845451103